radix-engine/src/system/system_modules/limits/module.rs

```diff
@@ -14,7 +14,7 @@ use crate::{
 };
 
 #[derive(Debug, Clone, PartialEq, Eq, ScryptoSbor)]
-pub enum TransactionLimitsError {
+pub enum LimitingError {
```

Comment:
Weird name? What is wrong with the original name?

Reply:
Wanted to be consistent with `CostingError`, but happy to change.